playground/server/api/pets.ts

```diff
@@ -0,0 +1,12 @@
+export default defineEventHandler(async (event) => {
+  const { $fetchPets } = useNuxtOpenFetchServer()
+  const data = await $fetchPets ("/pet/{petId}", {
+    path: {
+      petId: 1,
+    },
+  });
+
+  return ({
+    data,
+  })
+})
```

Comment:
Please lint this (get rid of 'no new line at the end of file'). The project uses [editorconfig](https://github.com/enkot/nuxt-open-fetch/blob/18c4ef73a106f95378fd4313ebeb1ef0c25de22a/.editorconfig#L9) but its setup was ignored.

Reply:
Thanks for the hint :+1: